### ISSUE TITLE
[chr4:14331587-14332165][INS][SVLEN=578bp]

### ISSUE THREAD

**Issue by jzook** [jzook/testHG008curation#173]
`chr4	14331587	Minda_103	N	<INS>	.	PASS	SVLEN=578;SVTYPE=INS;SUPP_VEC=ONT_severus_INS17703,ONT_Sniffles2.INS.DBM3,ONT_i_277,PB_severus_INS15603,PB_Sniffles2.INS.D5M3,PB_i_665
`

https://v2.genomeribbon.com/?session=https://42basepairs.com/download/s3/giab-data/ribbon-json/ribbon-hg008-cov-bedpe.json&locus=chr4:14331587#ribbon

**Comment by vkpat** [comment]
<img width="801" alt="Screenshot 2024-10-03 at 1 41 16 PM" src="https://github.com/user-attachments/assets/414eb7fc-45a9-4c2d-9719-d14636a9eee5">
<img width="984" alt="Screenshot 2024-10-03 at 1 42 28 PM" src="https://github.com/user-attachments/assets/1ce73527-3d38-46f5-b023-5c68e147de4a">
<img width="1706" alt="Screenshot 2024-10-03 at 1 43 53 PM" src="https://github.com/user-attachments/assets/6c9fa08f-9eaa-4ba7-b711-ef4d3813f0a8">
<img width="1718" alt="Screenshot 2024-10-03 at 1 50 49 PM" src="https://github.com/user-attachments/assets/dcceee03-3377-49dd-b31f-dfffcd28a1fb">
Looks like an insertion from chr4:14331587:14,331,601 that matches with Dragen call.. Breakpoints is not exact here..

**Comment by jzook** [comment]
Inside a LINE, and looks like a partial LINE insertion based on the BLAT of the insert, but might be an interesting one for @jo-mc to take a look at. I think the differences in length between methods have to do with different ways this complex variant could be represented, as well as noise in the poly-A tail.

**Comment by jo-mc** [comment]
Yes looks like a truncated Line1 insertion. There is also a small deletion AGATTTAAGTCTTT, likely as a result from the insertion. No target site duplication typical of line1, probably as a result of the deletion. This small deletion causing the breakpoints to vary for clipped reads, and obscuring likely exact breakpoints. This appears to also be **loss of heterozygosity** as both revio and illumina show insertion (and deletion) for every mapped read, and 50x coverage for pacbio, 99x for illumina? .  dragen vcf :  PR:SR   69,0:74,0       0,40:0,32
illumina, showing deletion and insertion in every read: (NOTE: using chm13v2 ref)
![image](https://github.com/user-attachments/assets/01292162-4afc-457b-8377-89f29d33002f)
PacBio revio
![image](https://github.com/user-attachments/assets/e7877394-723e-49a8-b0ef-f069944f73ff)
